nov-13-hardfork-spec.md

```diff
@@ -0,0 +1,38 @@
+# November 13th Bitcoin Cash Hardfork Technical Details
+
+Version 1.0, 2017-12-01
+
+## Summary
+ 
+When the median time past[1] of the most recent 11 blocks (MTP-11) is greater than or equal to UNIX timestamp 1510600000 Bitcoin Cash will execute a hardfork according to this specification.  It includes the following three protocol changes:
+
+* Enforcement of LOW_S signatures (BIP 0146)
+* Enforcement of NULLFAIL (BIP 0146)
+* A replacement for the emergency difficulty adjustment. The algorithm for the new difficulty adjustment is described below
+
+## Difficulty Adjustment Algorithm Description
+
+To calculate the difficulty of a given block (Bn+1), with an MTP-11[1] greater than or equal to the unix timestamp 1510600000, perform the following steps:
+
+1. Let Bn be the Nth block in a Bitcoin Cash Blockchain.
+1. Let Blast be the middle block of the [Bn, Bn-1, Bn-2] when sorted by timestamp.
+1. Let Bfirst be the middle block of the [Bn-144, Bn-145, Bn-146] when sorted by timestamp.
+1. Let the Timespan (TS) be equal to the difference in unix timestamps between Blast and Bfirst within the range [72 * 600, 288 * 600].  Values outside should be treated as their respective limits
+1. Let the Work Performed (W) be equal to the difference in chainwork between Blast  and Bfirst.
+1. Let the Projected Work (PW) be equal to (W * 600) / TS.
+1. Let Target (T) be equal to the (2^256 - PW) / PW.  This is calculated by taking the two’s complement of PW (-PW) and dividing it by PW (-PW / PW).
+1. The target difficulty for block Bn+1 is then equal to the lesser of T and 0x00000000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+
```

Comment:
Best if we can add some test Vectors.

Also, how does this work on testnet. Maybe some more explanations required.